13-TeV-examples/uproot_python/HZZAnalysis.ipynb

```diff
@@ -625,8 +636,8 @@
     "\n",
     "    for s in samples: # loop over samples\n",
     "        if s not in ['data', r'Signal ($m_H$ = 125 GeV)']: # if not data nor signal\n",
-    "            mc_x.append( data[s]['mllll'] ) # append to the list of Monte Carlo histogram entries\n",
-    "            mc_weights.append( data[s].totalWeight ) # append to the list of Monte Carlo weights\n",
+    "            mc_x.append( ak.to_numpy(data[s]['mllll']) ) # append to the list of Monte Carlo histogram entries\n",
+    "            mc_weights.append( ak.to_numpy(data[s].totalWeight) ) # append to the list of Monte Carlo weights\n",
```

Comment:
Currrently one needs to add `ak.to_numpy` calls before plotting from awkward arrays (even if they are flat). This might change in the future.